src/SurveyDesign.jl

```diff
@@ -2,13 +2,13 @@
 function print_short(x)
     # write floats in short form
     if isa(x[1], Float64)
-        x = round.(x, sigdigits = 3)
+        x = round.(x, sigdigits=3)
```

Comment:
I guess this is the formatter's doing. In that case, let's keep the style like this - without spaces around equal signs for argument values - _but_ this should be a different PR. Also, this needs to be done in all the files in order to keep consistency.

Reply:
Yep the formatter reduces the space around `=` signs